src/main/java/org/jenkinsci/plugins/nouvoladivecloud/NouvolaBuilder.java

```diff
@@ -194,33 +193,33 @@ public boolean perform(AbstractBuild build, Launcher launcher, BuildListener lis
 
         // listen for a callback
         if (!retURL.isEmpty()){
+            String jsonMsg = "";
             try{
                 boolean posted = false;
                 ServerSocket server = new ServerSocket(listenPort);
-		int timeout = 60; //timeout default is 60 minutes
-		if(!listenTimeOut.isEmpty())
-		    timeout = Integer.parseInt(listenTimeOut);
-		server.setSoTimeout(timeout * 60000);
+		        int timeout = 60; //timeout default is 60 minutes
+		        if(!listenTimeOut.isEmpty())
+		            timeout = Integer.parseInt(listenTimeOut);
+		        server.setSoTimeout(timeout * 60000);
```

Comment:
Yes. The file has mixed tabs and spaces. No change in functionality here. This has been tested.